src/app/query/sbtc/sbtc-limits.query.ts

```diff
@@ -18,10 +18,10 @@ export const defaultSbtcLimits = {
 };
 
 const sbtcLimitsResponseSchema = z.object({
-  pegCap: z.number(),
-  perDepositCap: z.number(),
-  perDepositMinimum: z.number(),
-  perWithdrawalCap: z.number(),
+  pegCap: z.nullable(z.number()),
```

Comment:
```suggestion
  pegCap: z.number().nullable(),
```

think better chained